airdrops/index_v1.json

```diff
@@ -128,6 +128,23 @@
                 "chain_id": 1,
                 "decimals": 18
             }
+        },
+        "starknet": {
+            "csv_path": "airdrops/starknet.csv",
+            "asset_identifier": "STRK",
+            "url": "https://provisions.starknet.io/",
+            "name": "STRK",
+            "icon": "starknet.svg",
+            "icon_path": "airdrops/icons/starknet.svg",
+            "cutoff_time": 1718841600,
+            "new_asset_data": {
+                "asset_type": "EVM_TOKEN",
+                "address": "0xe485E2f1bab389C08721B291f6b59780feC83Fd7",
+                "name": "Starknet",
+                "symbol": "STRK",
+                "chain_id": 1,
+                "decimals": 18
```

Comment:
it is in coingecko in preview mode. Can you add it? https://www.coingecko.com/en/coins/starknet